### ISSUE TITLE
[Auditory] Approach to content

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-people-use-web-videos#23]
Currently the Auditory script flows from explanation of "hard of hearing" and "deafness" on to the types of barriers and the accommodation needed for each barrier. Although not bad, that flow was a little hard to follow. Somewhat back and forth.

I am wondering if it would work better to have "hard of hearing" introduced, discuss accommodations that help that population. Then introduce "deafness" and the accommodations that help that population. Then move into basic auditory accommodations that help both populations, and everyone else.

Maybe there is an even better way to walk through the barriers and accommodations.

**Comment by nitedog** [comment]
Addressed in latest revision.
